tests/test_roxarapi/test_roxarapi_reek.py

```diff
@@ -329,6 +329,19 @@ def test_rox_get_modify_set_polygons(roxar_project):
     poly.to_roxar(roxar_project, "RESCALED", "", stype="clipboard")
     assert poly.dataframe.shape[0] == 127
 
+    # store and retrieve in general2d_data just to see that it works
+    if roxinstance.version_required("1.6"):
+        poly.to_roxar(roxar_project, "xxx", "folder/sub", stype="general2d_data")
+        poly2 = xtgeo.polygons_from_roxar(
+            roxar_project, "xxx", "folder/sub", stype="general2d_data"
+        )
+        assert poly.dataframe[poly.xname].sum() == pytest.approx(
```

Comment:
Here we can test all values with `poly.dataframe[poly.xname].values.tolist() == pytest.approx(poly2.dataframe[poly.xname])`